app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="container devise-container">
+  <h2>Sign up</h2>
+
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+      <%= f.label :name %><br />
+      <%= f.text_field :name %>
+    </div>
+    
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field">
```

Comment:
- [ ] Good job adding devise, however,  I will suggest you add an `input` for `bio` when we are signing up, to make sure we have data for the user's bio and to make sure we keep the wireframes:

Given

![Capture1](https://github.com/balatstar/blog/assets/57408419/008944bc-2bab-4fcd-9ca6-0469580e93e9)

Your project 

![Capture](https://github.com/balatstar/blog/assets/57408419/b2107e07-f9f1-4c23-8f8a-9bfb5bc06db5)